src/models/generative-model.test.ts

```diff
@@ -120,9 +123,7 @@ describe("GenerativeModel", () => {
           value.includes("be friendly") &&
           value.includes("temperature") &&
           value.includes("testField") &&
-          value.includes(HarmBlockThreshold.BLOCK_LOW_AND_ABOVE) &&
-          value.includes("presencePenalty") &&
-          value.includes("frequencyPenalty")
```

Comment:
These can be left in, and the two new ones added as well. The first set of checks in 98-101 only check that these properties were added to the model class. This set of checks is even more important as it ensures that when generateContent is called, these are actually passed in the body to makeModelRequest. This just searches the body text for any mention of the param, so it doesn't matter if it's nested - it's all just one big JSON string.